src/Kernel/ClassOrganization.class.st

```diff
@@ -305,7 +305,7 @@ ClassOrganization >> protocolsOfSelector: aSelector [
 	^ (self protocols select: [:each | each includesSelector: aSelector ]) asArray
 ]
 
-{ #category : #'backward compatibility' }
+{ #category : #removing }
```

Comment:
why protocol change?

Reply:
I moved it because it is used by a lot of things so I don't think it is just for backward compatibility